crypto/src/conversation.rs

```diff
@@ -329,11 +334,26 @@ impl MlsConversation {
         let message = match message {
             ProcessedMessage::ApplicationMessage(app_msg) => (Some(app_msg.into_bytes()), None),
             ProcessedMessage::ProposalMessage(proposal) => {
-                self.group.store_pending_proposal(*proposal);
+                if is_external_add_proposal(proposal.as_ref()) {
+                    let other_clients = self
+                        .group
+                        .members()
+                        .iter()
+                        .map(|kp| kp.credential().identity())
```

Comment:
credential identity is not a unique client identifier, hence there might be duplicates. Eliminate them